spec/smartling_pusher_spec.rb

```diff
@@ -66,7 +72,7 @@ def add_file_to_repo
     InMemoryDataStore::CommitLog.create(
       repo_name: repo_name,
       commit_id: commit_id,
-      phrase_count: 1,
+      phrase_count: nil,
```

Comment:
No, `nil` means the field hasn't been filled out yet (attempting to mimic ActiveRecord here).